snowdrop-execution/snowdrop-execution.cabal

```diff
@@ -55,6 +55,8 @@ library
       Snowdrop.Execution.DbActions
       Snowdrop.Execution.DbActions.Types
       Snowdrop.Execution.DbActions.Simple
+      Snowdrop.Execution.DbActions.Simple.Actions
```

Comment:
does it make sense to reflect in name that it's `STM` Actions?

Reply:
AVLp actions are also STM